manifests/common/config/connector/activemq/hosts_iteration.pp

```diff
@@ -6,7 +6,12 @@
     value => $middleware_hosts_array[$name - 1],
   }
 
-  $port = $mcollective::middleware_ssl ? {
+  $middleware_ssl = is_bool($mcollective::middleware_ssl) ? {
+    true  => $mcollective::middleware_ssl,
+    false => str2bool($mcollective::middleware_ssl),
```

Comment:
No need to use is_bool() here.  Since stdlib 4.0.0, str2bool also accepts something that's already a bool and just returns it.